pkg/util/kube/predicates.go

```diff
@@ -108,25 +108,30 @@ func (r *vmForVMClassPredicate) Generic(e event.GenericEvent) bool {
 func (r *vmForVMClassPredicate) matches(obj runtime.Object) (bool, error) {
 	obj = obj.DeepCopyObject() // do not mutate the original
 
-	vm, err := r.getVM(obj)
+	object, kind, err := r.getWatchedObj(obj)
```

Comment:
Please rename `object` to `obj` as the latter is clear and more succint.

Reply:
Let me know if you prefer other var name.